This uses parts of QtPlatformSupport which depend on QtDBus; this should
probably be fixed in qt5-base itself though.

--- origsrc/qt5ct-0.27/src/qt5ct-qtplugin/qt5ct-qtplugin.pro	2016-09-20 01:18:36.000000000 -0500
+++ src/qt5ct-0.27/src/qt5ct-qtplugin/qt5ct-qtplugin.pro	2016-10-30 01:30:19.714150100 -0500
@@ -4,7 +4,7 @@ TEMPLATE = lib
 TARGET = qt5ct
 CONFIG += plugin
 
-QT += gui-private platformsupport-private
+QT += gui-private dbus platformsupport-private
 
 SOURCES += \
     main.cpp \
